tests/pytest_tests/unit_tests/test_artifacts/test_wandb_artifacts.py

```diff
@@ -513,9 +513,6 @@ def test_unlogged_artifact_other_method_errors():
     with pytest.raises(ArtifactNotLoggedError, match="Artifact.get"):
         art["obj_name"]
 
-    with pytest.raises(ArtifactNotLoggedError, match="Artifact.link"):
```

Comment:
Good catch here - I just realized that `run.link_artifact()` auto-logs artifacts before linking!